configure.ac

```diff
@@ -190,7 +190,7 @@ if test "x$enable_werror" = "xyes"; then
   if test "x$CXXFLAG_WERROR" = "x"; then
     AC_MSG_ERROR("enable-werror set but -Werror is not usable")
   fi
-  AX_CHECK_COMPILE_FLAG([-Werror=vla],[ERROR_CXXFLAGS="$ERROR_CXXFLAGS -Werror=vla"],,[[$CXXFLAG_WERROR]])
```

Comment:
I assume you mean to remove this line? It looks out of place here since you already have it in the `test CXXFLAGS_overridden` block, but wanted to double check.

Reply:
Yes, this is intentional. This line enabled `-Werror` only for `-Wvla`, whereas we want it enabled for every warning. The line in the `test CXXFLAGS_overridden` block is what actually enables the `-Wvla` warning.